index.html

```diff
@@ -180,19 +262,21 @@ <h3 class="framework-text">Skills</h3>
                 <textarea rows="4" cols="50" maxlength="500" id="msj" placeholder="Enter text here" required></textarea>
                 <button class="form-buttom" type="submit">Get in touch</button>
```

Comment:
- [ ] It's a bit funny how your button seems to be running away from getting clicked. It keeps floating back and forth upon clicking even. Please kindly make it fixed at a spot, and work on implementing different styles depending on user's interaction. Here is a brief screencast of the button in action: 

[Screencast from 05-03-2023 00:44:32.webm](https://user-images.githubusercontent.com/99715640/222934526-4f95d8e9-e90f-4b56-90f9-8b6e1c7ca80b.webm)